src/aws/index.ts

```diff
@@ -34,7 +30,7 @@ const connectionCache = new KeyedResolvOnce<bs.Connection>();
 export const connect: ConnectFunction = (
   db: Database,
   remoteDbName = "",
-  url = "https://aws.amazon.com",
+  url = "aws://aws.amazon.com",
```

Comment:
one of the problems we face is that the external user who hasn't read our docs is likely to paste a URL from their world, and then our job in this function is to convert it to our configuration thing.  However that is sub optimal b/c it doesn't allow access to our full configuration space... I think this is an area that needs design in 0.20

Reply:
don't do it here than do it in the very todo top layer.
And I learned that if somebody don't read the docu you can not help him.
I have to think about the connect function it would love to remove it and join it with the gateway start